pkg/scheduler/framework/session_plugins.go

```diff
@@ -538,12 +539,15 @@ func (ssn *Session) JobOrderFn(l, r interface{}) bool {
 		}
 	}
 
-	// If no job order funcs, order job by CreationTimestamp first, then by UID.
+	// If no job order funcs, order job by CreationTimestamp first, then by Name.
 	lv := l.(*api.JobInfo)
 	rv := r.(*api.JobInfo)
 	if lv.CreationTimestamp.Equal(&rv.CreationTimestamp) {
-		return lv.UID < rv.UID
+		// Use the Name of the Job instead of UID to get deterministic order by Job name
+		klog.V(3).Infof("Creation timestamps are the same for job %v and %v: %v . using name to order", lv.Name, rv.Name, lv.CreationTimestamp)
+		return lv.Name < rv.Name
```

Comment:
If encounter a situation with the same name, that is, vcjobs with the same name that are not in the same namespace,  the JobOrder doesn't return the same consistent list order too?